connect/file/src/test/java/org/apache/kafka/connect/file/FileStreamSourceConnectorTest.java

```diff
@@ -55,18 +54,14 @@ public void setup() {
 
     @Test
     public void testConnectorConfigValidation() {
-        replayAll();
         List<ConfigValue> configValues = connector.config().validate(sourceProperties);
         for (ConfigValue val : configValues) {
             assertEquals(0, val.errorMessages().size(), "Config property errors: " + val.errorMessages());
         }
-        verifyAll();
```

Comment:
Don't we need any `verify` calls here?

Reply:
Thank you for the review, the only mock object in this test case is `ctx`, and the whole method has nothing to do with it, so I don't think we need verify here.